reference/views/macroeconomic_indices.view.lkml

```diff
@@ -0,0 +1,56 @@
+include: "//looker-hub/reference/views/macroeconomic_indices.view.lkml"
+
+view: +macroeconomic_indices {
+
+  label: "Macroeconomic indices"
+  fields_hidden_by_default: yes
+
+  dimension: symbol {
+    hidden: no
+    primary_key: yes
+  }
+
+  dimension_group: market {
+    hidden: no
+  }
+
+  measure: open_price {
+    hidden: no
+    type: average
+    sql: ${open};;
+    description: "Opening price for symbol on market_date."
+  }
+
+  measure: close_price {
+    hidden: no
+    type: average
+    sql: ${close};;
+    description: "Closing price for symbol on market_date."
+  }
+
+  measure: adj_close_price {
+    hidden: no
+    type: average
+    sql: ${adj_close};;
+    description: "Adjusted closing price for symbol on market_date."
+  }
+
+  measure: high_price {
+    type: max
```

Comment:
ah i forgot that when you hide by default you have to unhide the measures ... sorry that makes it only a slightly better solution

are those last three measures hidden by design or do you need to add the `hidden: no` line?